src/main/java/com/exasol/bucketfs/list/ListingRetriever.java

```diff
@@ -18,11 +19,14 @@
  * This class retrieves the list of buckets or of objects inside a bucket.
  */
 public class ListingRetriever {
+    /** Maximum time to wait for a list request */
+    public static final int LIST_TIMEOUT_SECONDS = 90;
```

Comment:
Store `Duration` here instead of int. Then you don't need to care about the unit (seconds, ms, ...)